app/views/content/life-as-a-teacher/explore-subjects/history/_article.html.erb

```diff
@@ -0,0 +1,73 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach history</h2>
+    <div class="inset">
+      <p>Teaching history, you’ll challenge pupils to scrutinise evidence to build a picture of the past. Guiding them through the often elusive sources left to us, you will help them question the reliability and accuracy of what they read. They’ll build skills in research, handling evidence, and creating arguments.</p>
+      <%= render 'content/shared/teaching/benefits' %>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_progression_chris' %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching history opens the door for your pupils to pursue exciting careers in fields such as research, curation, and law.</p>
+      <p>Your pupils will learn more from you than you might think. As their history teacher, you'll develop your pupils’ skills to investigate the past by interrogating primary and secondary sources. You'll teach them valuable life skills such as independent study, writing persuasively, and using evidence to back up points of view.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_history_jack' %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-history-programmes-of-study/national-curriculum-in-england-history-programmes-of-study">national curriculum for history</a>.</p>
+    <p>
+Key Stage 3 history aims to build on students' understanding of the history of Britain and the wider world. Some themes you may study include:</p>
```

Comment:
```suggestion
Key stage 3 history aims to build on students' understanding of the history of Britain and the wider world. Some themes you may study include:</p>
```

Reply:
Probably worth including the age groups here too, and like with the English subject page. So for consistency to keep it to age and then key stage in brackets.